tests/src/ClientTest.php

```diff
@@ -57,6 +58,7 @@ public function testContainer(): void
         $this->assertInstanceOf(FacetApiInterface::class, $container->get('facet'));
         $this->assertInstanceOf(TokenApiInterface::class, $container->get('token'));
         $this->assertInstanceOf(TokenAwareInterface::class, $container->get('textIngestion'));
+        $this->assertInstanceOf(TokenAwareInterface::class, $container->get('fileIngestion'));
```

Comment:
Lets add one more line to test FileIngestionApiInterface
```
        $this->assertInstanceOf(FileIngestionApiInterface::class, $container->get('fileIngestion'));
```
same for text and delete